### DISCUSSION TITLE
adding QuantizeLinear and DequantizeLinear to the project

### DISCUSSION THREAD

**Discussion by Soyuz0** [ENOT-AutoDL/onnx2torch#123] [Show and tell]
hello
I'm trying to a add a QuantizeLinear and DequantizeLinear layers and I just want someone to take a look because I'm having issues with them in my own project:

[Archive 2.zip](https://github.com/ENOT-AutoDL/onnx2torch/files/10474235/Archive.2.zip)

**Comment by Mirian-Hipolito** [comment]
Were you able to fix this issue? I am having similar problems with the implementation

**Comment by vasabi-root** [comment]
May be my [implementation](https://github.com/vasabi-root/onnx2torch/commit/a3362df4f11c1d0c8236387d7d3f89a7e250b595) will be useful. See the comment under the commit to understand the key implementation points.
